### PR TITLE
`operation-log.md`: Give a brief description of `@`, `+`, `-`

### PR DESCRIPTION
**Pull request by ilyagr**
This is based on a discussion in Discord where we realized this isn't documented.

### CODE REVIEW

File: docs/operation-log.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ The operation log allows you to undo an operation (`jj [op] undo`), which doesn'
 need to be the most recent one. It also lets you restore the entire repo to the
 way it looked at an earlier point (`jj op restore`).
 
+When referring to operations, you can use `@` to represent the current operation
+as well as `+` and `-` operators to get the descendant or an ancestor of an
```
Comment (martinvonz) [inline docs/operation-log.md:21]: `+` is not supported yet

Comment (martinvonz) [inline docs/operation-log.md:21]: `-` gets the parent operations, not all ancestors

**Review by martinvonz — APPROVED** [review]
Thanks for taking care of this

File: docs/operation-log.md (hunk 1 above)
Comment (ilyagr) [reply docs/operation-log.md:21]: Good point. I should've double-checked.

Comment (ilyagr) [reply docs/operation-log.md:21]: Also a good point.

**Pull request merged**